GameServerConsole/Program.cs

```diff
@@ -48,7 +46,7 @@ private static void Main(string[] args)
                     var startInfo = new ProcessStartInfo(leaguePath)
                     {
                         Arguments = String.Format("\"8394\" \"LoLLauncher.exe\" \"\" \"127.0.0.1 {0} {1} 1\"",
-                            parsedArgs.ServerPort, gameServerBlowFish),
+                            parsedArgs.ServerPort, "17BLOhi6KZsTtldTsizvHg=="),
```

Comment:
Don't you have the GameInfoJson to get this key from there?

Reply:
right, I shouldn't code when tired.